### ISSUE TITLE
Brodie - metadata update - linked samples

### ISSUE THREAD

**Issue by dehays** [microbiomedata/nmdc-metadata#253]
Using [Brodie_biosample_linking_update](https://docs.google.com/spreadsheets/d/1Gj6jwU5d8kdYhq8zuvjTB3T4-nt3Ru7NhxYpTeR9eRw/edit?usp=sharing) as input.

Update existing omics_processing documents. Iterate through the values in column A, using this dataset id as the reference to existing omics_processing objects (where "id" is "emsl:" + value in column A). Please take note of any case where no omics_processing object is found for a given EMSL dataset ID.

Verify that the "omics_type" object's "has_raw_value" matches the value in column B (Omics Type) We expect all these to match, but update the "omics_type" to the value from column B and log if they do not.

Update the value "has_input" to be an array of size == 1 where the one value is the IGSN biosample ID from column D in the form: "igsn:" + column D string

**Comment by dwinston** [comment]
The working notebook for this issue and for #252 is [here](https://github.com/microbiomedata/nmdc-metadata/blob/fa000e848ec4658b88aa564ac1804e896bd6d4de/metadata-translation/notebooks/ghissue_252_253_linked_samples.ipynb). #252 is done (pushed to dwinston_share db).

The notebook is a bit long because (1) I decided to use the Google Sheets/Drive APIs to source the spreadsheet and the json template, and (2) I needed to update the mongo collection $jsonSchema validator given recent updates to the NMDC JSON Schema to include previously missing fields for Biosample objects. I encountered no issues for #252 wrt unexpectedly present/missing documents or conflicting omics_type values.
